dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql

```diff
@@ -256,25 +256,13 @@ WITH
       FROM {{ source('zeroex_ethereum', 'ExchangeProxy_evt_RfqOrderFilled') }} fills
       LEFT JOIN {{ source('prices', 'usd') }} tp ON
           date_trunc('minute', evt_block_time) = tp.minute
-          {% if is_incremental() %}
```

Comment:
Why remove this? This will make the runtime much slower.

Reply:
np, added back